nautobot/core/tests/test_views.py

```diff
@@ -71,6 +79,34 @@ def test_search(self):
         response = self.client.get(f"{url}?{urllib.parse.urlencode(params)}")
         self.assertHttpStatus(response, 200)
 
+    def test_appropriate_models_included_in_global_search(self):
+        # Check for existing models in core apps and example app
+        existing_models = []
+        core_app_list = ["circuits", "dcim", "extras", "ipam", "tenancy", "virtualization"]
+        for model in apps.get_models():
+            if model._meta.app_label in core_app_list:
+                existing_models.append(f"{model._meta.model_name}")
```

Comment:
Similarly, could do `AppConfig` (`CircuitConfig`, etc.) `.get_models()` for each AppConfig to just get the models of that app, instead of iterating the global models list?